<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GLChangePassword</class>
 <widget class="QDialog" name="GLChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1109</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>GLChangePassword</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widgetNewAna" native="true">
   <property name="geometry">
    <rect>
     <x>538</x>
     <y>270</y>
     <width>844</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>0</y>
      <width>811</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>修改密码</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblBg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>844</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 119, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblType">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>150</y>
      <width>91</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPassword">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>205</width>
      <height>37</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editUser">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>205</width>
      <height>37</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>54</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>密码:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPassword2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>205</width>
      <height>37</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>101</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>确认密码:</string>
    </property>
   </widget>
   <zorder>lblBg</zorder>
   <zorder>lblTitle</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>lblType</zorder>
   <zorder>editPassword</zorder>
   <zorder>editUser</zorder>
   <zorder>label_7</zorder>
   <zorder>editPassword2</zorder>
   <zorder>label_8</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
